doc/main.dox

```diff
@@ -117,9 +117,9 @@ sockperf: [tid 4857] using select() to block on socket(s)
  -p      --port               	Listen on/connect to port <port> (default 11111).
  -m      --msg-size           	Use messages of size <size> bytes (minimum default 14).
  -f      --file               	Tread multiple ip+port combinations from file <file> 
-                                (server uses select).
+                                (server uses epoll).
```

Comment:
you may consider something general:
(server uses epoll) -> will use IO muxer such as select, poll, or epoll

BTW, do you know what is "Tread" here?